### PR TITLE
Descriptive Select

### PR DESCRIPTION
**Pull request by qkdreyer**
Issue: rgsystemes/kb#4960

### CODE REVIEW

File: src/Molecules/Editable/Editable.stories.jsx [hunk 1]
```diff
@@ -40,14 +38,14 @@ const CustomFormControl = ({
     <InputLabel>
       Label
       <FlexBox gap={1}>
-        <Select value={first} onChange={e => onChange([e.target.value, second])}>
+        <Select value={first} onChange={e => onChange([e.target.value, second])} native>
           {options.map(({ value, label }) =>
-            <option value={value}>{label}</option>,
+            <Option key={value} value={value}>{label}</Option>,
```
Comment (Beny13) [inline src/Molecules/Editable/Editable.stories.jsx:43]: Why did you need to add a `key`?

File: src/Molecules/Filter/Filter.stories.jsx [hunk 2]
```diff
@@ -28,8 +28,8 @@ export const filter = () => <Container>
   <Filter
     type="select"
     options={[
-      { name: '1', label: 'Choice 1' },
-      { name: '2', label: 'Choice 2' },
+      { value: '1', label: 'Choice 1' },
+      { value: '2', label: 'Choice 2' },
```
Comment (Beny13) [inline src/Molecules/Filter/Filter.stories.jsx:32]: What was the problem?
Can it happen on existing code using Filter?

File: src/Atoms/Select/Select.jsx [hunk 3]
```diff
@@ -1,14 +1,51 @@
...
+}))
+
+const inputStyles = makeStyles(theme => createStyles({
+  root: {
+    height:                  'auto !important', // FIXME: refacto input instead
+    borderBottomLeftRadius:  '0 !important', // FIXME: refacto input instead
+    borderBottomRightRadius: '0 !important', // FIXME: refacto input instead
+  },
+  input: {
+    padding: '6px 16px !important', // FIXME: refacto input instead
```
Comment (Beny13) [inline src/Atoms/Select/Select.jsx:27]: Should we ask mesteche about this?

File: src/Molecules/Editable/Editable.stories.jsx (hunk 1 above)
Comment (qkdreyer) [reply src/Molecules/Editable/Editable.stories.jsx:43]: https://reactjs.org/docs/lists-and-keys.html#keys

File: src/Molecules/Filter/Filter.stories.jsx (hunk 2 above)
Comment (qkdreyer) [reply src/Molecules/Filter/Filter.stories.jsx:32]: https://github.com/rgsystemes/lib-ui/blob/af2669ff9117434ce39343c7877004f3eb204b0c/src/Molecules/Filter/Filter.jsx#L75

It can happen if one is using the `Filter` component using `type="select"` with options not having `value` keys

File: src/Atoms/Select/Select.jsx (hunk 3 above)
Comment (qkdreyer) [reply src/Atoms/Select/Select.jsx:27]: He's the one who told me to add these `FIXME` (https://github.com/rgsystemes/lib-ui/commit/6fab4c42a94a0c721f71f7c9d6ee5b251b4b319d)
